### ISSUE TITLE
different behavior between Chronos Dongle and Yardstick One with 2FSK modulation

### ISSUE THREAD

**Issue by rfscan** [atlas0fd00m/rfcat#144]
I have an LRS pager, with rfcat + yardstick one fw ver:  0606,  also I have an Chronos Dongle with firmware 0631.  

d.setMdmModulation(MOD_2FSK)
d.setFreq(446156250)
d.setMdmSyncMode(0)
d.setMdmDeviatn(4800)
d.setMdmDRate(625)
d.setMaxPower()
d.setEnableMdmManchester(1)

everything works fine with Yardstick, but the LRS pager does not accept a signal from Chronos Dongle with the same rfcat settings

**Comment by atlas0fd00m** [comment]
that's really weird.  can you get a capture of it (both transmissions) with an SDR?

**Comment by rfscan** [comment]
yes of course, here is an archive with two transmissions from the URH
[hackrf-rfcat.zip](https://github.com/atlas0fd00m/rfcat/files/13060627/hackrf-rfcat.zip)

**Comment by atlas0fd00m** [comment]
i see the format (complex16s), but what's the samplerate, or anything else i might need to pull these up in InSpectrum (or another tool you recommend)?

**Comment by atlas0fd00m** [comment]
from initial glance, it looks like the capture on the chronos isn't as clean as the ys1 (specifically, there's no dead-space at the front), so any attempts to line up transmissions is nigh impossible.  can you send one with dead space and possible pre-amble for both?

**Comment by rfscan** [comment]
Hey Atlas,

here are two new dumps (now they captured with hackrf_transfer tool). 
Each dump contains three submissions - RFxmit(bytes.fromhex("aaaaaafc2d00002b00000000000156"))
Hackrf_transfer was run before each RFxmit execution. 

chronos:~/chronos$ hackrf_transfer -f 446156250 -w
Receive wav file: HackRF_20240131_105929Z_446156kHz_IQ.wav
call hackrf_set_sample_rate(10000000 Hz/10.000 MHz)
call hackrf_set_hw_sync_mode(0)
call hackrf_set_freq(446156250 Hz/446.156 MHz)

I hope this helps, have a nice day!

[chronos-hackrf.zip](https://github.com/atlas0fd00m/rfcat/files/14110168/chronos-hackrf.zip)
[ys1-hackrf.zip](https://github.com/atlas0fd00m/rfcat/files/14110189/ys1-hackrf.zip)
